udata/migrations/2016-04-05-deactivate-datapublica.js

```diff
@@ -0,0 +1,22 @@
+/*
+ * Deactivate all datasets linking to data-publica
+ */
+
+var eurostatId = ObjectId('534fff76a3a7292c64a77ded');
+var banqueMondialeId = ObjectId('534fff59a3a7292c64a77cff');
+
+var datasets = db.dataset.find({
+    $or: [
+        {organization: eurostatId},
+        {organization: banqueMondialeId }
+    ]
+});
+var count = 0;
+
+datasets.map(function(dataset) {
```

Comment:
You can avoid looping over all the dataset by performing an `update()` instead of a `find()`:
```javascript
var result = db.dataset.update(<query>, {$set: {deleted: new Date()}}, {multi: true});
print(result.nbModified, 'datasets deleted');
```